corehq/apps/users/validation.py

```diff
@@ -6,7 +6,7 @@
 from corehq.apps.users.views.mobile import BAD_MOBILE_USERNAME_REGEX
 
 
-def validate_mobile_username(username, domain):
+def validate_mobile_username(username, domain, is_unique=True):
```

Comment:
Could you add `is_unique` to the docstring below? Same in `generate_mobile_username` too.